src/sentry/taskworker/retry.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from collections.abc import Sequence
+from multiprocessing.context import TimeoutError
+
+from sentry_protos.sentry.v1alpha.taskworker_pb2 import RetryState
```

Comment:
```suggestion
from sentry_protos.sentry.v1.taskworker_pb2 import RetryState
```

For when the proto release is done.